interface/forms/vitals/report.php

```diff
@@ -26,125 +27,440 @@ function US_weight($pounds, $mode = 1)
-                } else { //pulse and respirations
-                    $vitals .= "<td><div class='font-weight-bold d-inline-block'>" . xlt($key) . ": </div><div class='text' style='display:inline-block'>" . text($value) . " " . xlt('per min') . "</div></td>";
-                }
-            } else {
-                $vitals .= "<td><div class='font-weight-bold d-inline-block'>" . xlt($key) . ": </div><div class='text' style='display:inline-block'>" . text($value) . "</div></td>";
-            }
-
-            $count++;
-
-            if ($count == $cols) {
-                $count = 0;
-                $vitals .= "</tr><tr>\n";
-            }
+
+    //***Vitals Add
+    $ageYMD = getPatientAgeYMD($patient_data['DOB'], $data['date']);
+    $ageDays = ceil(getPatientAgeInDays($patient_data['DOB'], $data['date']));
+    $age_in_months = $ageYMD['age_in_months'];
+
+    //Check if patient has Down Syndrome.  If so, we use the CDC charts since WHO does not have
+    //data available.
+    $hasDS = hasDownSyndrome($data['pid']);
+
+
+    //else print years and months
+    $data["Age: "] = $ageYMD['ageinYMD'];
+    $data["Age in Days: "] = $ageDays;
+    $data["BMI"] = number_format(($data['weight'] / 2.204) / pow(($data['height'] * 2.54 * .01),2),2) ;
+    if ($ageDays <= 1856 && !$hasDS) {
+        $who_data = GrowthCharts::getWhoStats($ageDays, $patient_data['sex'], $data['weight'] / 2.204, $data['height'] * 2.54, $data['head_circ'] * 2.54);
+        $data["Height %ile WHO"] = number_format($who_data['height_pct'], 2) ?? "-";
+        $data["Height Zscore WHO"] = number_format($who_data['height_Z']['Zscore'], 2) ?? "-";
+        $data["Weight %ile WHO"] = number_format($who_data['weight_pct'], 2) ?? "-";
+        $data["Weight Zscore WHO"] = number_format($who_data['weight_Z']['Zscore'], 2) ?? "-";
+        $data["Weight for Height %ile WHO"] = number_format($who_data['weight_height_pct'], 2) ?? "-";
+        $data["Head Circ %ile WHO"] = number_format($who_data['head_pct'], 2) ?? "-";
+        $data["Head Circ Zscore WHO"] = number_format($who_data['head_Z']['Zscore'], 2) ?? "-";
+        $data["BMI %ile WHO"] = number_format($who_data['bmi_pct'], 2) ?? "-";
+        $data["BMI Zscore WHO"] = number_format($who_data['bmi_Z']['Zscore'], 2) ?? "-";
+
+    }
+
+    if (($age_in_months >= 23.5) && ($age_in_months < 240.5) && !$hasDS) {
+        $cdc_data = GrowthCharts::getCdcStats($age_in_months, $patient_data['sex'], $data['weight'] / 2.204, $data['height'] * 2.54, $data['BMI']);
+        $data["Height %ile CDC"] = number_format($cdc_data['height_pct'], 1 ?? 'x');
+        $data["Weight %ile CDC"] = number_format($cdc_data['weight_pct'], 1) ?? 'x';
+        $data["Weight for Height %ile CDC"] = number_format($cdc_data['weight_height_pct'], 1) ?? 'x';
+    }
+
+    //currently data is unavailable
+    if ($age_in_months < 24 && $hasDS) {
```

Comment:
These next two if statements are needed or not needed? can drop the if statement if they're not needed.

Reply:
I left those in there for when I can find datasets (raw data, in .csv) for patients with Down Syndrome.  I can remove them but I kept them in there for future purposes.

Reply:
If they are for the future just comment them out and/or leave a reminder for the future (or future developers of what you are intending to do here.  I don't see the point in having executable code that does nothing that will break if someone removes a variable or refactors things in the future.

Reply:
Good point.  I'll go ahead and do that.